dash/SQLConnection.py

```diff
@@ -1,11 +1,14 @@
 import sqlite3
 import pandas as pd
+import threading
+
+lock = threading.Lock()
```

Comment:
this is neat, didn't know about this. Have you added `threading` to the `requirements.txt`?

Reply:
threading is a python module, so it doesn't need to be in there, the requirements.txt just installs 3rd party modules